app/src/main/java/app/accrescent/client/ui/AppList.kt

```diff
@@ -33,6 +35,7 @@ import app.accrescent.client.R
 import app.accrescent.client.data.InstallStatus
 import kotlinx.coroutines.launch
 
+@SuppressLint("FrequentlyChangedStateReadInComposition")
```

Comment:
I'm not sure how best to handle this issue, but I don't think we should ignore it because observing `listState.firstVisibleItemScrollOffset` directly could cause a lot of recompositions.

Reply:
I might have an idea, I'll come back to you shortly.

Reply:
Updated now with a better solution. Unfortunately we can't save the exact scroll state that way if we're at the exact top of the app list, because of the animation when opening an app's screen, because it scrolls the whole list to the top. If we didn't ignore that case, we would always save 0 as position and 0 as offset, so it wouldn't work.